content/zh-tw/posts/eth2-staking-with-k8s-prysm.md

```diff
@@ -0,0 +1,619 @@
+    ```
+
+    這個指令會持續監控狀態變化，我們只需等到 beacon 及 validator 用戶端都變成 Running 狀態即可。
+
+2. 檢查 beacon 的執行記錄
+
+    ```bash
+    microk8s kubectl logs -f -nprysm -l app=beacon
+    ```
+
+3. 檢查 validator 的執行記錄
+
+    ```bash
+    microk8s kubectl logs -f -nprysm -l app=validator1
+    ```
+
+    如果想檢查其他的 validator，可以將`-l app=<validator name>`更改成在`values.yaml`設定的其他 validator 的名字，以 validator2 為例
+
+    ```bash
+    microk8s kubectl logs -f -nprysm -l app=validator2
+    ```
+
+### 使用 Helm Chart 更新 Prysm 版本
+
+以太坊 2.0 用戶端的新版本推出速度很快，我們應該盡快更新用戶端版本來獲得最新的bug fixes和功能。為了簡化版本跟軟體部署的管理，我們推薦用 Helm 來更新版本：
+
+1. 到 [Prysm Github 版本釋出頁面](https://github.com/prysmaticlabs/prysm/releases)查看最新版本
+
+2. 將`values.yaml`中的 **image.version** 改成最新版本，例如 `v1.3.4`
+
+3. 儲存`values.yaml`之後執行以下 Helm 指令更新用戶端
+
+    ```bash
+    microk8s helm3 upgrade eth2xk8s ./eth2prysm -nprysm
+    ```
+
+4. 檢查部署設定，確認用戶端已更新成新版本
+
+    ```bash
+    microk8s helm3 get manifest eth2xk8s -nprysm
+    ```
+
+5. 依照[檢查用戶端狀態](#檢查用戶端狀態)章節檢查用戶端是否正常執行
+
+### 使用 Helm 回溯版本
+
+如果版本回溯不牽涉資料庫 schema 變動的話，使用 Helm 回溯版本就跟更新一樣直覺。以下是範例步驟及指令：
+
+1. 使用`helm history`指令找出並記下想要回溯到的版本號碼
+
+    ```bash
+    microk8s helm3 history eth2xk8s -nprysm
+    ```
+
+2. 回溯到指定版本（以下指令假設我們要回溯到版本 4）
+
+    ```bash
+    microk8s helm3 rollback eth2xk8s 4 -nprysm
+    ```
+
+3. 接著依照「[檢查用戶端狀態](#檢查用戶端狀態)」章節檢查部署設定以及用戶端是否正常執行。
+
+如果版本回溯前需要先還原資料庫 schema，可以參照「[使用 Helm 回溯版本（如果資料庫 Schema 有更動)](#使用-helm-回溯版本如果資料庫-schema-有更動)」章節。
+
+某些情況下有可能沒辦法回溯到之前的版本（[例子](https://docs.prylabs.network/docs/prysm-usage/staying-up-to-date/#downgrading-between-major-version-bumps)），在回溯前記得先確認用戶端相關文件。
+
+## 結論
+
+感謝你的閱讀！我們希望這篇文章能夠幫助想要使用 Kubernetes 來做以太坊 2.0 staking 的你。我們會繼續製作相關教學，之後我們也會開發給其他以太坊 2.0 用戶端的 Helm Chart。敬請期待！
+
+## 有任何建議或是疑問嗎？
+
+請讓我們知道你的想法！如果對這篇文章有任何問題及建議都歡迎到我們[網站的 Github 專案](https://github.com/lumostone/lumostone.github.io)開 issue 或是發 pull request。如果你對於貢獻以太坊 2.0 Staking 的 Helm Chart 有興趣，請將 issue 或 pull request 發至 [eth2xk8s](https://github.com/lumostone/eth2xk8s) 跟我們聯繫。
+
+## 附錄
+
+### 檢查 CPU 及記憶體用量
+
+如果想知道每個 pod 的 CPU 及記憶體用量， 可以使用 Github 開源專案 [metrics server](https://github.com/kubernetes-sigs/metrics-server) 來取得資料。
+
+1. 首先，用以下的指令在 Kubernetes 叢集上安裝 metrics server
+
+    ```bash
+    microk8s kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+    ```
+
+2. 接著可以執行 `kubectl top` 指令來得到用量資訊，下面兩個例子分別會得到 beacon  validator 的用量。
+
+    ```bash
+    microk8s kubectl top pod -l app=beacon
+    microk8s kubectl top pod -l app=validator
+    ```
+
+### 解除安裝 Helm Chart
+
+如果想要停止執行以及移除 Prysm，可以執行以下指令來移除整個 Helm Chart ：
+
+```bash
+microk8s helm3 uninstall eth2xk8s -nprysm
+```
+
+### 使用 Helm 回溯版本(如果資料庫 Schema 有更動)
+
+以 [Prysm v1.3.0 版本](https://github.com/prysmaticlabs/prysm/releases/tag/v1.3.0)為例，如果想要回溯至 v1.2.x，我們需要在回溯前先跑一個腳本來還原 v1.3.0 帶來的資料庫 schema 變動。所以如果我們照著「[使用 Helm 回溯版本](#使用-helm-回溯版本) 」的步驟執行指令，在用 Helm 改變 Prsym 版本成 v1.2.2 之後，所有的 pods 會馬上重啟，但 Prysm 可能會因為資料庫 schema 只適用於 v1.3.0 版本而無法正常啟動。
```

Comment:
It's not about this PR but can you remove `for example` in `Take Prysm v1.3.0 release for example as an example` in the EN version?